lib/resources/r4/appointment_patch.yaml

```diff
@@ -63,6 +63,32 @@ operations:
         <li>CodeableConcept.text is required and CodeableConcept.coding, if present, will be ignored</li>
         <li>When the value array is empty, reasonCode will be removed</li>
       </ul>
+  - name: add-cancelationReason
```

Comment:
Do we typically order the patch operations supported by the order of the field listed in the resource spec?

Reply:
Ignore for now. Other operations are out of order. someone can change them all at a later time if it's needed.